app/models/hubstats/comment.rb

```diff
@@ -8,7 +8,7 @@ class Comment < ActiveRecord::Base
       .group("hubstats_comments.user_id")
     }
 
-    scope :created_since, lambda {|time| where("created_at > ?", time) }
+    scope :created_in_date_range, lambda {|start_date, end_date| where("hubstats_comments.created_at > ? AND hubstats_comments.created_at < ?", start_date, end_date)}
```

Comment:
You might want to use >= and <= to include things created exactly at the boundaries.